.github/workflows/ci.yaml

```
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+    
+  rubocop:
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+
+  test:
+    name: Tests
```

The tests in Kafo are quite slow (especially the acceptance tests). Here I'd make the tests depend on RuboCop. That way you avoid running them if there are simple mistakes.
```suggestion
    name: Tests
    needs: rubocop
```

For gems where the tests are very fast I generally don't do that, but here it's worth it.